src/main/java/org/prebid/server/auction/PrivacyEnforcementService.java

```diff
@@ -59,12 +59,12 @@ public PrivacyEnforcementService(GdprService gdprService, BidderCatalog bidderCa
      */
     Future<Map<String, PrivacyEnforcementResult>> mask(Map<String, User> bidderToUser, ExtUser extUser,
                                                        List<String> bidders, Map<String, String> aliases,
-                                                       BidRequest bidRequest, String publisherId, Timeout timeout) {
+                                                       BidRequest bidRequest, Boolean isGdprEnforced, Timeout timeout) {
```

Comment:
Rename to `isGdprEnforcedByAccount` since we have several similar flags in `PrivacyEnforcementService`.
Pls change name through all flow.